packages/api-request-builder/src/build-query-string.js

```diff
@@ -73,7 +75,9 @@ export default function buildQueryString(
       queryString.push(`offset=${offsetParam}`)
     }
     if (sort && sort.length)
-      queryString = queryString.concat(sort.map(s => `sort=${s}`))
+      queryString = queryString.concat(
+        sort.map((s: string): string => `sort=${s}`)
```

Comment:
In theory this should not be necessary because it's already typed in `ServiceBuilderDefaultParams`.
I'm wondering though if by destructuring it loses the type def.

Can you try removing the destructurings and see if it's correctly typed?